plinth/modules/tor/templates/tor.html

```diff
@@ -53,7 +53,11 @@
       {% endif %}
     </p>
 
-    {% include "diagnostics_button.html" with module="tor" %}
+    {% if status.is_running %}
+      {% include "diagnostics_button.html" with module="tor" %}
```

Comment:
I have not considered the users module usage.  We can have `enabled=` parameter rather than `app_is_running=` parameter to make it sound less awkward when an app in involved.

Perhaps something like this:

In case of regular module usage:
```
{% include "diagnostics_button.html" with module=diagnostics_module_name enabled=service.is_running %}
```
In case of users_list.html it will become:
```
{% include "diagnostics_button.html" with module="users" enabled="True" %}
```

I am also not very particular :) If you think current way is okay, I will just merge.